modules/dr-restoring-cluster-state.adoc

```diff
@@ -97,10 +97,49 @@ The output of this command should be empty. If it is not empty, wait a few minut
 $ sudo mv /var/lib/etcd/ /tmp
 ----
 
+.. If the `/etc/kubernetes/manifests/keepalived.yaml` file exists, follow these steps:
+
+... Move the `/etc/kubernetes/manifests/keepalived.yaml` file out of the kubelet manifest directory:
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/keepalived.yaml /tmp
+----
+
+... Verify that any containers managed by Keepalived are stopped:
++
+[source,terminal]
+----
+$ sudo crictl ps --name keepalived
+----
++
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
+
+... Check if the control plane has any Virtual IPs (VIPs) assigned to it:
++
+[source,terminal]
+----
+$ ip -o address | egrep '<api_vip>|<ingress_vip>'
```

Comment:
Here is using 'egrep' while below is using 'grep -E', they're equivalent (searching for an extended regular epression), but probably should be the same command.
I might suggest sticking to 'egrep' as it looks like other parts of this documentation make use of it.